### PR TITLE
Fix status error

### PR DESCRIPTION
**Pull request by clairecng**
## What does this PR do?

- [ ] Adds new functionality
- [ ] Alters existing functionality
- [x] Fixes a bug
- [ ] Improves documentation or testing

Please briefly describe your changes as well as the motivation behind them:
- Wrong error log when the status is and empty field

## Code Quality Checklist

- [ ] The documentation is up to date.
- [ ] My code is sufficiently commented and passes continuous integration checks.
- [ ] I have signed my commit (see [Contributing Docs](../CONTRIBUTING.md)).

## Testing

- [ ] I leveraged continuous integration testing
    - [ ] by depending on existing `unit` tests or `end-to-end` tests.
    - [ ] by adding new `unit` tests or `end-to-end` tests.
- [ ] I manually tested the following steps:
    - `x`
    - [ ] locally.
    - [ ] as a canary deployment to a cluster.

### CODE REVIEW

File: controllers/disruption_controller.go [hunk 1]
```diff
@@ -386,7 +386,7 @@ func (r *DisruptionReconciler) updateInjectionStatus(instance *chaosv1beta1.Disr
 
 	terminationStatus := disruptionTerminationStatus(*instance, chaosPods)
 	if terminationStatus != tsNotTerminated {
-		switch instance.Status.InjectionStatus {
+		switch status {
```
Comment (clairecng) [inline controllers/disruption_controller.go:389]: is there a reason I'm not seeing for why we're not using the status variable?

Comment (luphaz) [reply controllers/disruption_controller.go:389]: Instance status is updated by the method `disruptionTerminationStatus` hence the variable value won't be the same IMO

**Review by luphaz — APPROVED** [review]

**Review by aymericDD — APPROVED** [review]
LGTM 👍

**Review by taihuynh167 — APPROVED** [review]

**Comment by ptnapoleon** [comment]
@clairecng this was closed because it was fixed in a separate PR, no? I remember seeing the same diff elsewhere

**Comment by clairecng** [comment]
> @clairecng this was closed because it was fixed in a separate PR, no? I remember seeing the same diff elsewhere

@ptnapoleon yes this was fixed!

**Pull request closed without merging**